onyo/lib/command_utils.py

```diff
@@ -76,16 +77,16 @@ def fill_unset(assets: Generator[dict, None, None] | filter,
 def natural_sort(assets: list[dict],
                  keys: list[str] | None = None,
                  reverse: bool = False) -> list[dict]:
-    r"""Sort an asset list by a given list of `keys`.
+    r"""Sort an asset list by a list of `keys`.
```

Comment:
Not sure about the rule anymore, but is "keys" with one backtick correct, or should it have two, like above in the doc-string of `def fill_unset()`?